site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -1791,7 +1791,17 @@ public function ajaxSaveGradedComponent($gradeable_id) {
             $this->core->getOutput()->renderJsonFail('Insufficient permissions to save component/marks');
             return;
         }
-        //don't allow peer graders to save custom marks
+
+
+        //don't allow custom marks if they are disabled
+        if ($custom_message != null || $custom_points != null) {
+            if (!$gradeable->getAllowCustomMarks()) {
+                $this->core->getOutput()->renderJsonFail('Custom marks are disabled for this assignment');
+                return;
+            }
+        }
+
+        //don't allow peer graders to save custom marks no matter hoe gradeable is configured
```

Comment:
hoe -> how

Reply:
I wasn't able to find where custom marks are stored in the data base so I couldn't write a check for it. I instead checked if grading had begun and displayed the warning. If that's not satisfactory I can try to come up with something but I just wasted a ton of time trying to do a work around that didn't end up working.

Reply:
Update - Chris helped me find it, it wasn't showing up in my data base because I'm an idiot, I'll fix soon.